service-gateway-mock/docker/Dockerfile-mocksirius-ci

```diff
@@ -0,0 +1,19 @@
+FROM stoplight/prism:4
```

Comment:
https://github.com/ministryofjustice/opg-pdf-service/blob/main/Dockerfile this may help with creating this as an alpine container and installing dependencies

Reply:
⏫